homepage.html

```diff
@@ -30,79 +31,195 @@
         margin: 0;
         padding: 0;
       }
+
+       /*set the background*/
+       .bg {
+        background: url('./img/homepage_img.jpg');
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover; 
+        /* vertical-align: middle; */
+        /* background-clip:border-box; */
+        height:auto;
+        line-height: auto;
+        background-color:transparent;
+
+      }
+      .inner{
+        height:100%;
+        background: rgba(138, 134, 134, 0.575);
+      }
+
+      
+      .marginSearch{
+        margin-bottom:5%;
+      }
+      .input-group {
+            width: 35%;
+        }
+
+        .textBgLf{
+        background:rgb(252, 250, 250);
+        height:95%;
+        width:80%;
+        /* text-align: center; */
+        margin-left: 7%;
+        /* margin-right:1%; */
+        margin-top:0%;
+        margin-bottom:1%;
+        font-size:100%;
+        color:rgb(17, 17, 17);
+        border-radius:25px;
+        padding:0.5cm;
+      }
+
+      .textBgRt{
+        background:rgb(252, 250, 250);
+        height:95%;
+        width:80%;
+        /* text-align: center; */
+        /* margin-left: 1%; */
+        margin-right:1%;
+        margin-top:0%;
+        margin-bottom:1%;
+        font-size:100%;
+        color:rgb(17, 17, 17);
+        border-radius:25px;
+        padding:0.5cm;
+      }
+      .tag{
+        width: auto;
+        height: 180%;
+        /* object-fit:cover; */
+
+      }
+
     </style>
+    <link href="css/style.css" rel="stylesheet" />
+
+    <link rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
+    <script src="https://cdn.staticfile.org/jquery/2.1.1/jquery.min.js"></script>
+    <script src="https://cdn.staticfile.org/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+   
+
+   
   </head>
   <body onload="explore()">
-    <div class="container-fluid">
-      <div class="row">
-        <div class="col-md-12">
-          <h3>
-            Home Page
-          </h3>
-          <ul class="nav">
-            <li class="nav-item">
-              <a class="nav-link active" href="addroom.html">Add Room</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="signup.html">Sign up</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="index.html">Sign Out</a>
-            </li>
-          </ul>
-        </div>
-      </div>
-      <div class="row">
+      <nav class="navbar navbar-default navbar-fixed-top" role="navigation"> 
+          <div class="container-fluid"> 
+            <div class="navbar-header">
+              <div class="navbar-brand">
+                <img  src="http://ubuntu1604-006.student.cs.uwaterloo.ca:19490/img/logo.png" class="tag" alt="" >
```

Comment:
I used this image url because I had to use the website layoutit and using this full url was the only way to display this logo there, however for our own website we can simply use `/img/logo.png` as src.